lib/LANraragi/Plugin/Metadata/Eze.pm

```diff
@@ -115,6 +128,30 @@ sub tags_from_eze_json {
     my $site   = $hash->{"gallery_info"}->{"source"}->{"site"};
     my $gid    = $hash->{"gallery_info"}->{"source"}->{"gid"};
     my $gtoken = $hash->{"gallery_info"}->{"source"}->{"token"};
+    my $category = $hash->{"gallery_info"}->{"category"};
+    my $uploader = $hash->{"gallery_info_full"}->{"uploader"};
+    my $timestamp = $hash->{"gallery_info_full"}->{"date_uploaded"};
+
+    if ( $timestamp ) {
+        # convert microsecond to second
+        $timestamp = $timestamp / 1000;
+    } else {
+        my $upload_date = $hash->{"gallery_info"}->{"upload_date"};
+        my $time = timegm_posix($$upload_date[5],$$upload_date[4],$$upload_date[3],$$upload_date[2],$$upload_date[1]-1,$$upload_date[0]-1900);
```

Comment:
Use an absolute 4-digit year here, we don't need the rolling behavior imo:  
> Years greater than 999 are interpreted as being the actual year, rather than the offset from 1900. Thus, 1964 would indicate the year Martin Luther King won the Nobel prize, not the year 3864.

```suggestion
        my $time = timegm_modern($$upload_date[5],$$upload_date[4],$$upload_date[3],$$upload_date[2],$$upload_date[1]-1,$$upload_date[0]);
```